app/src/main/java/org/mozilla/focus/autocomplete/UrlAutoCompleteFilter.java

```diff
@@ -6,124 +6,27 @@
 package org.mozilla.focus.autocomplete;
 
 import android.content.Context;
-import android.content.res.AssetManager;
-import android.content.res.Resources;
-import android.os.AsyncTask;
-import android.support.annotation.VisibleForTesting;
-import android.util.Log;
+import android.text.TextUtils;
 
-import org.mozilla.focus.locale.Locales;
 import org.mozilla.focus.widget.InlineAutocompleteEditText;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.nio.charset.StandardCharsets;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.Locale;
-import java.util.Set;
+import mozilla.components.browser.domains.DomainAutoCompleteProvider;
 
 public class UrlAutoCompleteFilter implements InlineAutocompleteEditText.OnFilterListener {
```

Comment:
You may be interested in using `ui-autocomplete` eventually. This will replace `InlineAutocompleteEditText`. For one the version component will replace `OnFilterListener` with a simple function (yay!) and more importantly we landed some bug fixes! :)

Reply:
Will do it in another PR. Thanks!